src/include/miopen/conv/asm_implicit_gemm.hpp

```diff
@@ -225,6 +225,43 @@ HeuristicInitMacroTileNoPadGemmK(size_t gemm_m,
         return std::make_tuple(0, 0, 0);
 }
 
+// this is to support big tensor > 4G. need to decide how many splits needed
+// return the number of splits
```

Comment:
```suggestion
// This is to support big tensor > 4G. Need to decide how many splits needed.
// Return the number of splits.
```
Ditto other new comments.